### ISSUE TITLE
Favicon in IE9 not presented

### ISSUE THREAD

**Issue by CaptionF** [owncloud/core#18012]
### Steps to reproduce
1. install oC 8.0.5
2. use IE9 
3. check Favicon on the open browser tab

### Expected behaviour
Favicon should show ownCloud icon
see image below (from IE11)
![favicon_missing_ie9-3](https://cloud.githubusercontent.com/assets/10551219/9011554/4279f05e-37ae-11e5-9a6b-b2c6ebfca812.jpg)
:

### Actual behaviour
IE icon visible on IE 9

![favicon_missing_ie9-2](https://cloud.githubusercontent.com/assets/10551219/9011594/799d7286-37ae-11e5-9540-01f684ebe053.jpg)

### Server configuration
RHEL 6.6

**Web server:**
Apache 2.4
**Database:**
Mysql
**PHP version:**
5.4.
**ownCloud version:** (see ownCloud admin page)
oc8.0.5
**Updated from an older ownCloud or fresh install:**
fresh

### Client configuration
IE 9

**Operating system:**
Win 7

**Comment by karlitschek** [comment]
@jancborchardt any idea?

**Comment by MorrisJobke** [comment]
We currently support favicon in PNG format: 
```
<link rel="shortcut icon" type="image/png" href="/master/core/img/favicon.png">
```

But IE9 needs an 16x16 pixel ico file of mimetype `image/x-icon`.

**Comment by MorrisJobke** [comment]
IE 11+ only support PNG favicons: http://caniuse.com/#feat=link-icon-png

**Comment by MorrisJobke** [comment]
See also my comment about what we need to do to fix this in IE. It's a lot of stuff and hard to address in generic way for all environments. I doubt that it is worth the effort.

**Comment by MorrisJobke** [comment]
https://github.com/owncloud/core/pull/18039#issuecomment-132788154

**Comment by PVince81** [comment]
Tested IE8, IE9, IE10, IE11, Firefox and Chrome, combined with login page, public link and regular page => favicon works :+1:

**Comment by MorrisJobke** [comment]
@CaptionF This will be fixed in 9.0.0
